crates/bevy_core/src/time/time.rs

```diff
@@ -47,3 +81,60 @@ impl Time {
 pub(crate) fn time_system(mut time: ResMut<Time>) {
     time.update();
 }
+
+#[cfg(test)]
+mod tests {
+    use super::Time;
+    use bevy_utils::{Duration, Instant};
+
+    #[test]
+    fn update_test() {
+        let start_instant = Instant::now();
+
+        // Create a `Time` for testing
+        let mut time = Time {
+            startup: start_instant,
+            ..Default::default()
+        };
+
+        // Ensure `time` was constructed correctly
+        assert_eq!(time.delta, Duration::from_secs(0));
```

Comment:
Where possible lets test the public getters instead of the private fields to ensure we're exercising the public api. Not super important when its just a direct passthrough, but that could change in the future.

Reply:
Good idea, I wrote the tests before changing the API so I completely missed that!